File: Grind75/Week1/TwoSum.java
class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> numToIndex = new HashMap<>();
        for(int i = 0; i < nums.length; i++){
            if(numToIndex.containsKey(target-nums[i])){
                return new int[] {numToIndex.get(target-nums[i]), i};
            }
            numToIndex.put(nums[i], i);
        }
        return null;
    }
}

File: Grind75/Week1/TwoSum.py
class TwoSum:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict = {}
        for i in range(len(nums)):
            comp = target - nums[i]
            if comp in dict:
                return [dict[comp], i]
            else:
                dict[nums[i]] = i
        return []
